app-crypt/ssdeep/ssdeep-2.14.1-r1.ebuild

```diff
@@ -12,33 +12,21 @@ SRC_URI="https://github.com/${PN}-project/${PN}/releases/download/release-${PV}/
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv x86"
-IUSE="static-libs"
-
-DOCS=(
-	AUTHORS
-	ChangeLog
-	FILEFORMAT
-	NEWS
-	README
-	TODO
-)
 
 PATCHES=(
 	"${FILESDIR}/${PN}-2.10-shared.patch"
 )
 
 src_prepare() {
 	default
+
 	eautoreconf
 }
 
-multilib_src_configure() {
-	ECONF_SOURCE="${S}" \
-		econf \
-			$(use_enable static-libs static)
-}
+src_install() {
```

Comment:
Maan, that's a lot of empty files for such a tiny function.

Reply:
I prefer the blank lines, but w/e. Dropped.